pkg/scalers/kafka_scaler.go

```diff
@@ -594,6 +594,9 @@ func (s *kafkaScaler) getTopicPartitions() (map[string][]int32, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error describing topics: %w", err)
 	}
+	s.logger.Info(
+		fmt.Sprintf("with topic name %s the list of topic metadata is %v", topicsToDescribe, topicsMetadata),
+	)
```

Comment:
As it's a debug logger, let's change the verbosity:
```suggestion
	s.logger.V(1).Info(
		fmt.Sprintf("with topic name %s the list of topic metadata is %v", topicsToDescribe, topicsMetadata),
	)
```
WDYT @zroubalik ?

Reply:
Agree!  Could you please add this to the other kafka scaler that we have? (If possible)

Reply:
sure. I was searching for something like `s.logger.Debug()` but could not find any. Thanks for the suggestion

Reply:
@zroubalik  for apache_kafka_scaler, I can see that they have put similar log. Do you want to put the logging anywhere else ? https://github.com/kedacore/keda/blob/4cbc74e51738eae35f9c8f476bdb3bc7fea03982/pkg/scalers/apache_kafka_scaler.go#L423-L429

Reply:
Just make sure we expose the same info when needed and also in the same log level (I can see 4 in this snippet, so make it 1 pls)

Reply:
I have made changes on the apache_kafka_scaler too!